hacks/cutNewVersions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash   
+
+# Before running this script make sure these versions are correct!
+NEW_DOCS_VERSION="8.1"
+PREVIOUS_DOCS_VERSION="8.0"
+NEW_OPTIMIZE_VERSION="3.9.0"
+PREVIOUS_OPTIMIZE_VERSION="3.8.0"
+
+
+echo $NEW_DOCS_VERSION
+echo $PREVIOUS_DOCS_VERSION
+echo $NEW_OPTIMIZE_VERSION
+echo $PREVIOUS_OPTIMIZE_VERSION
+
+
+echo "1/ Creating version $NEW_DOCS_VERSION of main docs..."
+npm run docusaurus docs:version $NEW_DOCS_VERSION
+
+echo "2/ Creating version $NEW_OPTIMIZE_VERSION of Optimize docs..."
+npm run docusaurus docs:version:optimize $NEW_OPTIMIZE_VERSION
+
+echo "3/ Correcting sidebars files of new versions..."
+sed -i '' 's/\"href\": \"\/optimize\/next\//\"href\": \"\/optimize\//g' versioned_sidebars/version-$NEW_DOCS_VERSION-sidebars.json
+sed -i '' 's/\"href\": \"\/docs\/next\//\"href\": \"\/docs\//g' optimize_versioned_sidebars/version-$NEW_OPTIMIZE_VERSION-sidebars.json
+
+echo "4/ Correcting sidebars files of previous versions..."
+sed -i '' "s/\"href\": \"\/optimize\//\"href\": \"\/optimize\/${PREVIOUS_OPTIMIZE_VERSION}\//g" versioned_sidebars/version-$PREVIOUS_DOCS_VERSION-sidebars.json
+sed -i '' "s/\"href\": \"\/docs\//\"href\": \"\/docs\/${PREVIOUS_DOCS_VERSION}\//g" optimize_versioned_sidebars/version-$PREVIOUS_OPTIMIZE_VERSION-sidebars.json
```

Comment:
if it's useful, I can add descriptions about what each of these are doing. High-level they're correcting the URLs that get generated for cross-instance links, which all will change in both the new and previous versions. (Due to the fact that docusaurus puts versions in the URLs of old versions, but no versions in the current one.)

Reply:
Given that these are not things we need to modify regularly (if at all) I think I'm ok with not adding a further description.